M2/Macaulay2/d/gmp1.d

```diff
@@ -1,7 +1,11 @@
 -- this file is small, so the exported definitions can be imported into the engine
 declarations "
     #include <math.h>
+    #if USING_MPIR 
+    #include <mpir.h>
+    #else
     #include <gmp.h>
+    #endif
     #include <mpfr.h>
```

Comment:
I'd favor concentrating this chunk to only one place in d/ and perhaps one place in e/. What do you think?

Reply:
Me, too.   That chunk is all over the place.  (We haven't even tried using mpir with the eigen branch, by the way -- that will probably need some debugging.)

Done in commit 3ff184f79bf3daab27a74376c091ac1c394fbcec

Thanks.